internal/pkg/levee/propagation/propagation.go

```diff
@@ -36,6 +40,8 @@ type Propagation struct {
 	taggedFields fieldtags.ResultType
 }
 
+// Dfs performs a depth-first search of the graph formed by SSA Referrers and
+// Operands relationships.
```

Comment:
Consider adding `beginning at the root node n` to the doc comment to distinguish this exported `Dfs` as the entrypoint for a given root vs the recursive step `dfs` below.

Reply:
Done.